react/src/components/atoms/forms/InputCheckBox/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Input from '../Input';
+import Error from '../Input/error';
+import Icon from '../../icons/Icon';
+import { InputContext } from '../Input/context';
+import './style.css';
+
+const CheckBox = (props) => (
+  <React.Fragment>
```

Comment:
do we want to add a higher level label to this component since it could be used solo bolo?